config/data_sync_list/common.json

```diff
@@ -0,0 +1,12 @@
+{
+    "RetryAttempts": 3,
```

Comment:
Is there a way RetryAttempts and RetryInterval could have a good default?  I don't think we want everyone to have to decide these values, and instead just trust that the daemon will do the right thing.  especially because there's no reason for it to fail in the first place aside from something catastrophic so just a hiccup should be automatically handled.

Reply:
Had a re-thinking on this design. Also , defining retryAttempts and retryInterval on each JSON file will create conflicts. But defining a default value in daemon will loose the flexibility for different vendors for defining their default retry attempts if they wants. So thinking about an approach of defining a build option in meson and can be included the same values as macros in auto generated configuration file (config.h) .